pkg/security/security_profile/activity_tree/activity_tree.go

```diff
@@ -493,7 +494,7 @@ func (at *ActivityTree) buildBranchAndLookupCookies(entry *model.ProcessCacheEnt
 		if cs.isSet() {
 			fastMatch, found = at.CookieToProcessNode.Get(cs)
 			if found {
-				fastMatch.applyImageTagOnLineageIfNeeded(imageTag)
+				fastMatch.applyImageTagOnLineageIfNeeded(imageTag, fastMatch.Process.ExecTime)
```

Comment:
I would remove the second argument as the Process.ExecTime file is part of fastMatch